<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Photo ID</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>20</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Show the test image</string>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineImageURI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushImageSelector">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Open the file browser and select folder to process.</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="List">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>125</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Naming convention</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkOcrSegregate">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>80</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Separate uncertainly named images, otherwise ignore them.</string>
      </property>
      <property name="text">
       <string>Segregate uncertain images</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>271</width>
        <height>27</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Naming convention to apply to renamed files.</string>
      </property>
      <item>
       <property name="text">
        <string>SURNAME, FORENAME (GROUP X)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SURNAME, FORENAME</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>155</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Format:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineOutputURI">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Location of renamed image files.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushImageSelector_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Open the file browser and select an output folder.</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>45</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Output folder:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>15</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Advanced Options</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>45</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hough transform threshold:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinHoughThreshold">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string>Adjust to fine tune skew straightening effectiveness. Default: 65.</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>65</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>80</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hough transform maximum gap width:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinHoughGapPercentage">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>75</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string>Adjust to fine tune skew straightening effectiveness. Default: 10%</string>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>15</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Skew correction</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
